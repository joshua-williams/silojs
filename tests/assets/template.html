<!DOCTYPE html>
<html lang="en">
    <head>

    </head>

    <body>
        <aside>
            ${include('')}
        </aside>

        <section>

        </section>
    </body>
</html>