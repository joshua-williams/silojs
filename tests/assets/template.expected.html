<html lang="en">
    <head>

    </head>

    <body>
        ${partial()
    </body>
</html>